opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/WebTracingConfiguration.java

```diff
@@ -22,9 +22,20 @@ public static Builder builder() {
         return new Builder();
     }
 
+    public Builder toBuilder() {
```

Comment:
We should also document that this is safe to use only with bean post processing (or maybe some similar approaches) because config values are propagated to different parts.